py3status/modules/mega_sync.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+"""
+Display status of MEGA sync.
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module (default 10)
+    format: display format for this module (default "MEGA: {SyncState_0}")
+
+Format placeholders:
+    <column>_<ID> where <column> and <ID> are returned by 'mega-sync' command.
+
+Requires:
+    MEGAcmd: command-line interface for MEGA
+
+@author Maxim Baz (https://github.com/maximbaz)
+@license BSD
+
+SAMPLE OUTPUT
+{'color': '#00FF00', 'full_text': 'MEGA: Synced'}
+"""
+
+
+class Py3status:
+    """
+    """
+
+    # available configuration parameters
+    cache_timeout = 10
+    format = "MEGA: {SyncState_0}"
+
+    def post_config_hook(self):
+        if not self.py3.check_commands("mega-sync"):
+            raise Exception("MEGAcmd is not installed")
```

Comment:
I was thinking about naming it `mega`, but then there are a bunch of other tools in MEGAcmd (`mega-backup`, `mega-speedlimit`, `mega-webdav`, etc.), I can easily envision people creating more py3status modules for those specific actions. And I don't think one huge module would be a good fit for all these tools, what they do is very very different. So let's call it `megasync` then.

Reply:
I'm with `mega_sync`, `megasync`, `mega`. Just wanted to point this out. :+1:

Reply:
OK, extracted the constant

Reply:
>but then there are a bunch of other tools in MEGAcmd (mega-backup, mega-speedlimit, mega-webdav, etc.), I can easily envision people creating more py3status modules for those specific actions. And I don't think one huge module would be a good fit for all these tools,

If this is true... then we could name it `mega` because if you're making `format_sync` for your `mega-sync`, the other tools can be `format_speedlimit`, `format_backup`, `format_webdev`, etc... just the same.

Users would have to specify what they want to use in the `format` just same as what you're working right now... to put `{format_sync}` in `format`  instead of numbered sync placeholders.

Reply:
Now it makes sense, renamed again